presto-docs/src/main/sphinx/security/ldap.rst

```diff
@@ -0,0 +1,239 @@
+coordinator based on their group memberships. This optional feature is enabled
+if you set ``authentication.ldap.group-dn`` property.
+Only users belonging to the group with distinguished name
+``authentication.ldap.group-dn`` are authenticated successfully.
+
+In addition to the basic LDAP authentication properties, you need group-membership specific
+properties in ``config.properties``. Sample entries for both Active Directory and OpenLDAP are
+listed below.
+
+.. code-block:: none
+
+    authentication.ldap.base-dn=OU=America,DC=corp,DC=domain,DC=com
+    authentication.ldap.group-dn=CN=AuthorizedGroup,OU=America,DC=corp,DC=domain,DC=com
+    authentication.ldap.user-object-class=person
+
+.. note::
+
+    Presto does not yet support providing multiple groups for 
+    ``authentication.ldap.group-dn``. Nested group authorization, where you
+    want to authorize a user who belongs to a group which is a member of
+    ``authentication.ldap.group-dn`` is also not supported.
+
+======================================================= ======================================================
+Property                                                Description
+======================================================= ======================================================
+``authentication.ldap.base-dn``                         The base LDAP distinguished name for the user who
+                                                        who tries to connect to the server.
+``authentication.ldap.group-dn``                        The entire LDAP :abbr:`dn (distinguished name)` for
+                                                        the group which must be authorized to access Presto.
+                                                        The user trying to connect to the server with base
+                                                        :abbr:`dn (distinguished name)`
+                                                        ``authentication.ldap.base-dn`` must belong to this
+                                                        group for successful authentication.
+``authentication.ldap.user-object-class``               The LDAP objectClass the user implements in LDAP.
+                                                        Eg: ``person``.
+======================================================= ======================================================
+
+
+.. _cli_ldap:
+
+Presto CLI
+----------
+
+Environment Configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+TLS Configuration
+~~~~~~~~~~~~~~~~~
+
+Access to the Presto coordinator should be through HTTPS when using LDAP
+authentication. The Presto CLI can use either a :ref:`Java Keystore
+<server_java_keystore>` file or :ref:`Java Truststore <cli_java_truststore>`
+for its TLS configuration. 
+
+If using keystore file it can be copied to the client machine and used
+for its configuration. If using truststore you can either use default
```

Comment:
Not sure what you mean by "for its configuration" ? Also, add a comma to make it look like: "If you are using truststore, you can".